src/inverseRobot.js

```diff
@@ -23,8 +23,21 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const valueObj = Object.values(robot);
+  const keyObj = Object.keys(robot);
+  const copy = {};
+
+  for (let i = 0; i < valueObj.length; i++) {
```

Comment:
much easier that way
```suggestion
  const copy = {};

  for (const key in robot) {
```